packages/mobile/src/invite/saga.ts

```diff
@@ -192,6 +210,9 @@ export function* sendInvite(
       Logger.error(TAG, 'Currently only dollar escrow payments are allowed')
     }
 
+    const e164NumberToAddress = { [e164Number]: null } // null means the number is unverified
```

Comment:
This would replace any existing mappings a user may have for this number, which we don't want. Instead, do this:
`yield put(updateE164PhoneNumberAddresses({}, addressToE164Number))`

Reply:
I see. Is this true because we don't have any logic to stop someone from inviting an existing user? Or is there a use case I am missing where a user might want to invite someone with a number already in the mapping?

Reply:
Someone could invite an existing user via the Invite screen from the Account/Settings menu. And more broadly, there's no reason for us wipe away any existing mappings that may exist for this number.

Reply:
Got it